staging/targets/trusted_root.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
I may be missing some context here, but this is not the latest trusted root (the TSA cert chain is missing). This might be the purpose though :)

Reply:
@kommendorkapten Is there an instance of the TSA in a GitHub staging environment? There's the values in prod which I could reuse, but I'd assume we would prefer a different instance.

Reply:
Yes it is. Let me find the certificate chain for that.

Reply:
@kommendorkapten Did you get a chance to find it? Otherwise I'll leave it blank for now.

Reply:
@haydentherapper sorry, was busy last Friday and was off until today (Public holiday here). Give me until the end of this day (Tuesday) and I should have it for you.

Reply:
@haydentherapper we are about to generate new staging keys this week, you want to wait for that or go on with an empty set for now?

Reply:
If I get a chance to this week I might continue with the generation just to make sure I have the steps, but in all likelihood I'll probably just wait til next week. Once you have the chain, can you either post it here or in an issue?

Reply:
I'll create an issue once we have the updated chain 👍